client/src/main/proto/spine/base/command.proto

```diff
@@ -79,13 +79,23 @@ message CommandContext {
     // The `namespace` attribute must be defined for commands in multitenant applications.
     Namespace namespace = 5;
 
-    oneof time_opts {
-        // The delay to wait before sending the command.
-        google.protobuf.Duration delay = 6;
+    // The command scheduling options.
+    Schedule schedule = 6;
+}
+
+message Schedule {
+    oneof options {
+        // The amount of time between the current moment and a command delivering to the target.
```

Comment:
`the current moment` isn't clear. Is this a moment at the client, or the moment of receiving the command at the server?